src/scales/scale.logarithmic.js

```diff
@@ -48,11 +48,14 @@ module.exports = function(Chart) {
 						if (valuesPerType[meta.type] === undefined) {
 							valuesPerType[meta.type] = [];
 						}
-
+                        
 						helpers.each(dataset.data, function(rawValue, index) {
 							var values = valuesPerType[meta.type];
 							var value = +me.getRightValue(rawValue);
-							if (isNaN(value) || meta.data[index].hidden) {
+                            if(value === 0){
+                                dataset.data[index] = null;  // don't show y-value
```

Comment:
Seems I found the solution. For implementation, we need to decide one point: what is the correct differance should be between 0 and the min value in the set that is !== 0 (for example 1)? Formaly, it is -infinity, but we cann't show this in the chart.